### PR TITLE
[R] Fix the R interface

### PR DESCRIPTION
**Pull request by tqchen**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -219,7 +211,7 @@ PSLITE:
 $(DMLC_CORE)/libdmlc.a: DMLCCORE
 
 DMLCCORE:
-	+ cd $(DMLC_CORE); make libdmlc.a USE_SSE=$(USE_SSE) config=$(ROOTDIR)/$(config); cd $(ROOTDIR)
```
Comment (piiswrong) [inline Makefile]: I think @vchuravy added this for power arch compile

Comment (vchuravy) [inline Makefile]: Yes we will need to pass the flag to the DMLC_CORE make file. I did not want to do architecture resolution in multiple places

**Comment by piiswrong** [comment]
Can we put code generation in make?

File: Makefile (hunk 1 above)
Comment (tqchen) [inline Makefile]: was a change mistakenly introduced by trying to use copy paste instead of merging, will resolve this

Comment (tqchen) [inline Makefile]: removed

**Comment by tqchen** [comment]
Code generation in make will be resolved in a separate PR https://github.com/dmlc/mxnet/issues/3343

**Comment by piiswrong** [comment]
any updates?

**Comment by tqchen** [comment]
Rebased and tested locally

**Comment by tqchen** [comment]
@piiswrong R test now passes

**Pull request merged**